pkg/interface/src/views/apps/permalinks/embed.tsx

```diff
@@ -131,8 +154,8 @@ function GraphPermalink(
         navigate(e);
       }}
     >
-      {loading && Placeholder()}
-      {showTransclusion && index && (
+      {loading && association && Placeholder(association.metadata.config.graph)}
```

Comment:
You failed tsc because of this though! Try `(association.metadata.config as GraphConfig).graph` and import `GraphConfig` type from `@urbit/api`